### PR TITLE
Javascript Folder: Uploading Assignment1 &2

### PR DESCRIPTION
**Pull request by Gopal32**

### CODE REVIEW

File: Gopal-Kothari/Javascript/Assignment1/Ques1.html [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+                var tem = x;
+                x = y;
+                y = tem;
+                document.getElementById("demo1").innerHTML = "<b> x = " + x +"<b>, y = " +y;
+            }
+            //without Temporary variable
+            function method2(){
+                var x = document.getElementById('x').value;
+                var y = document.getElementById('y').value;
+                [x,y] = [y,x];
```
Comment (Snehal-113) [inline Gopal-Kothari/Javascript/Assignment1/Ques1.html:38]: do it without arrays
x=x+y
y=x-y
x=x-y

File: Gopal-Kothari/Javascript/Assignment2/Ques1.html [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+        </center>
+        <script type="text/javascript">
+            // second largest number
+            function submit() {
+                var a = document.getElementById('a').value;
+                var b = document.getElementById('b').value;
+                var c = document.getElementById('c').value;
+                var d = document.getElementById('d').value;
+                var e = document.getElementById('e').value;
+                var arr = [a,b,c,d,e];
```
Comment (Snehal-113) [inline Gopal-Kothari/Javascript/Assignment2/Ques1.html:30]: try to do this program without arrays

**Review by Snehal-113 — CHANGES_REQUESTED** [review]

File: Gopal-Kothari/Javascript/Assignment1/Ques1.html (hunk 1 above)
Comment (Gopal32) [reply Gopal-Kothari/Javascript/Assignment1/Ques1.html]: done

File: Gopal-Kothari/Javascript/Assignment2/Ques1.html (hunk 2 above)
Comment (Gopal32) [reply Gopal-Kothari/Javascript/Assignment2/Ques1.html]: done

File: Gopal-Kothari/Javascript/Assignment3/Ques5.html [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+            <label>Enter number:</label>
+            <input type="number" id="a" /><br><br>
+            <button onclick="submit()">Submit</button>
+            <p id="demo"></p>
+        </center>
+        <script type="text/javascript">
+            // Factorial number
+            function submit() {
+                var a = document.getElementById('a').value;
+                var temp = "1";
```
Comment (Snehal-113) [inline Gopal-Kothari/Javascript/Assignment3/Ques5.html:20]: var temp=1;
1 is not a string

**Review by Snehal-113 — CHANGES_REQUESTED** [review]

**Comment by Gopal32** [comment]
close

**Pull request closed without merging**